File: Backend/src/main/java/com/tarikturkdil/dto/MyAccountResponse.java
package com.tarikturkdil.dto;

import lombok.Data;


//Sadece "benim profilim" endpoint'i (/api/users/me gibi)
@Data
public class MyAccountResponse extends DtoBase{

	private String username;
    private String email;
    private String avatarUrl;
    private String bio;
}

File: Frontend/app/src/main/java/com/tarikturkdil/photoProject/data/remote/dto/MyAccountResponse.kt
package com.tarikturkdil.photoProject.data.remote.dto

import kotlinx.serialization.Serializable

@Serializable
data class MyAccountResponse(
    val id: Long,
    val username: String,
    val email: String? = null,
    val avatarUrl: String? = null,
    val bio: String? = null,
    val createTime: String? = null
)